Resources/Prototypes/Atmospherics/reactions.yml

```
@@ -209,7 +209,272 @@
   - 0     # healium
   - 0     # nitrium
   effects:
-  - !type:PluoxiumProductionReaction {}
+  - !type:PluoxiumTritiumProductionReaction {}
+
+- type: gasReaction
+  id: MetalHydrogenReaction
+  priority: 1
+  maximumTemperature: 273.2
+  minimumRequirements:
+  - 0     # oxygen
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)